wp-seo-main.php

```diff
@@ -202,7 +202,8 @@ function _wpseo_activate() {
 	WPSEO_Options::ensure_options_exist();
 
 	if ( is_multisite() && ms_is_switched() ) {
```

Comment:
This if..else block can be shortened I think?

```php
if (multisite & switched){
  if (stripcategorybase){
    new WPSEO_rewrite;
  }
  add_action( 'shutdown', 'flush_rewrite_rules' );
}
```

Reply:
Yes, but that makes makes more sense to me, what do you think?
```php
$is_multisite_deactivation = is_multisite() && ms_is_switched();

if ( !$is_multisite_deactivation && WPSEO_Options::get( 'stripcategorybase' ) === true ) {
   // Constructor has side effects so this registers all hooks.
   $GLOBALS['wpseo_rewrite'] = new WPSEO_Rewrite();
}

add_action( 'shutdown', 'flush_rewrite_rules' );
```